Sources/MapboxNavigation/InstructionsCardViewController.swift

```diff
@@ -131,7 +134,9 @@ open class InstructionsCardViewController: UIViewController {
                   return nil
               }
         
-        return cell.subviews.compactMap({ $0 as? InstructionsCardContainerView }).first
+        let instructionsCardContainerView = cell.subviews.compactMap({ $0 as? InstructionsCardContainerView }).first
+        instructionsCardContainerView?.delegate = self
```

Comment:
Let the `InstructionCardViewController` as the delegate to each instruction label it contained, so there'll be only one `spriteRepository` stored in active navigation.